atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -253,7 +252,19 @@ public void compactInternally() {
                         + " IOT tables to compensate for bloat. You can contact Palantir Support if you'd"
                         + " like more information. Underlying error was: {}", tableRef, e.getMessage());
             } catch (TableMappingNotFoundException e) {
-                throw Throwables.propagate(e);
+                throw new RuntimeException(e);
+            }
+        } else {
+            try {
+                log.warn("Tried to clean up {} bloat after a sweep operation, but couldn't, because enterprise "
+                        + "features are disabled. "
+                        + "This means that even if you are running sweep, you will need to manually clean up your "
+                        + "tables, otherwise disk usage will continually increase. To free up space, run the "
```

Comment:
`will continually increase` has some caveats, you can reuse space if operating on a single table. i.e. you write many versions to a table, sweep it and you can write more versions reusing the deleted space.